### ISSUE TITLE
Andrid app Icon

### ISSUE THREAD

**Issue by jrahma** [dotnet/maui#2643]
Hi,

Please bring a UI to the Android app icon just like the iOS so developer can just click and upload the image file.

Thanks,
Jassim

**Comment by jamesmontemagno** [comment]
App Icons can not be defined directly in shared code and the single project -> https://github.com/dotnet/maui/blob/main/src/Templates/src/templates/maui-mobile/MauiApp.1.csproj#L32-L36

**Comment by jrahma** [comment]
What I was referring to is to have a UI for it just like the info.plist

regardless what android setup is, Visual Studio can show the UI to pick and select the images for the App Icon then VS will do the rest at the background.

This makes developer life easier.

**Comment by Redth** [comment]
@ThisIsKirsch - FYI this is basically the VS MauiAppIcon designer idea we've talked about in the past

**Comment by Zhanglirong-Winnie** [comment]
Verified this issue with Visual Studio Enterprise 17.9.0 Preview 1.0. Can repro on android platform.

**Comment by jfversluis** [comment]
I don't think we will be adding this any time soon. Also, this is something that should be reporting through VS Feedback as this is a tooling request. Thanks for the suggestion!
